lib/tasks/heroku.rake

```diff
@@ -1,19 +1,14 @@
 namespace :heroku do
   desc 'Heroku release task - runs on every code push, runs before postdeploy task'
   task release: :environment do
-    if ActiveRecord::Base.connection.schema_migration.table_exists?
-      Rake::Task['db:migrate'].invoke
-      Rake::Task['db:seed'].invoke
-      Rake::Task['curriculum:content:import'].invoke
-    else
-      Rails.logger.info 'Database not initialized, skipping database migration.'
-    end
+    Rake::Task['db:prepare'].invoke
+    Rake::Task['db:migrate'].invoke
+    Rake::Task['db:seed'].invoke
+    Rake::Task['curriculum:content:import'].invoke
   end
 
   desc 'Heroku postdeploy task - runs only on review app creation, after release task'
   task postdeploy: :environment do
-    Rake::Task['db:schema:load'].invoke
-    Rake::Task['db:seed'].invoke
-    Rake::Task['curriculum:content:import'].invoke
+    # No op
```

Comment:
This gave me grief again like the Rails 7.1 upgrade. I guess they just don't want that API to be public.

I think doing it like this should behave the same way as `db:prepare` is designed to be idemptotent - if the schema exists already it will do nothing.

The only downside is that seeds wil run twice on initial review app creation, but I don't _think_ it's an issue. I couldn't see any issues running the seed command locally multiple times.

Reply:
Nice! this looks like a much better way of doing it. 
Running seeds twice should be fine, they've been designed to be idempotent.

It should only affect review apps, and those appear to be working the same, so I think this task can go 💥